### PR TITLE
Joseph's PR for fetching data for Session Summary

### PR DESCRIPTION
**Pull request by JosephPark259**
This is what I have so far for the data that will be passed in to the Session Summary page.

For now my next TODO is:
1. finish countMessagesBySession in collection/chat.js
2. Add extra info on the Session Summary page ( speaker name on title, date&time, ...)
3. Add extra relevant fields to collection to finish the implementation for chat.js and questions.js TODOs

If there is anything I should add to my TODO list or anything to fix the code, let me know! (Don't worry about being nit-picky haha)

### CODE REVIEW

**Comment by JosephPark259** [comment]
On a side note: This isn't ready to be merged but just to be treated as a code review.

File: client/src/pages/SessionData.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Stats from '../components/Stats';
+
+class SessionData extends React.Component {
```
Comment (DevelopmentByDavid) [inline client/src/pages/SessionData.jsx]: would you mind converting this to a functional component?

File: client/src/components/Stats.jsx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+   },
+   paperSpacing: {
+      padding: '20px'
+   },
+   barWrapper: {
+      height: '400px'
+   }
+ }));
+
+function Stats({sent, asked, unanswered, duration}) {
```
Comment (DevelopmentByDavid) [inline client/src/components/Stats.jsx]: Add [prop types](https://reactjs.org/docs/typechecking-with-proptypes.html).  Do this for all your components.

File: client/src/pages/SessionData.jsx (hunk 1 above)
Comment (JosephPark259) [reply client/src/pages/SessionData.jsx]: Done!

**Comment by JosephPark259** [comment]
Update:
I've added a route of findSummary/:sessionId. I just decided to do this for now so we don't over fetch with the find/:sessionId. I couldn't get the aggregate to work so I just decided to use the usual way of querying. For now this implementation works with finding the total amount of questions per sessionId.

next TODO is to implement the route.post for the new session data that we talked about (start,end,duration...).

Let me know if you have anything in mind!

**Pull request merged**